src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -989,7 +989,7 @@ namespace winrt::TerminalApp::implementation
 
                 for (auto&& [profileIndex, remainingProfile] : remainingProfilesEntry.Profiles())
                 {
-                    items.push_back(_CreateNewTabFlyoutProfile(remainingProfile, profileIndex));
+                    items.push_back(_CreateNewTabFlyoutProfile(remainingProfile, profileIndex, winrt::hstring{}));
```

Comment:
nit: you may be able to get away with `{}` instead of `winrt::hstring{}` unless this function is templated. Maybe. I'm not actually sure.

Reply:
You're right and that is cleaner, changed: dd41243ed174c3df35939a983008e806b4e48055